dll/cpl/mmsys/lang/pt-PT.rc

```diff
@@ -31,7 +39,7 @@ BEGIN
     LTEXT "&Esquema de sons:", -1, 8, 42, 150, 9
     COMBOBOX IDC_SOUND_SCHEME, 8, 53, 230, 46, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     PUSHBUTTON "&Guardar como...", IDC_SAVEAS_BTN, 125, 70, 59, 15
-    PUSHBUTTON "&Apagar", IDC_DELETE_BTN, 188, 70, 50, 15
+    PUSHBUTTON "Eli&minar", IDC_DELETE_BTN, 188, 70, 50, 15
     LTEXT "Para modificar sons, clique num evento de programa na seguinte lista e, em seguida, seleccione um som a aplicar. Pode guardar as alterações como um novo esquema de som.", -1, 8, 90, 230, 27
     LTEXT "Eventos de &programas:", -1, 8, 118, 150, 9
```

Comment:
Unrelated, but you might want to cover also the "&programas" vs. "&Procurar..." collision in this dialog. Just a recommendation, since you do touch the file anyway. This was not introduced by you. It was wrong beforehand already.